tembo-cli/src/cmd/validate.rs

```diff
@@ -49,10 +49,19 @@ pub fn execute(verbose: bool) -> Result<(), anyhow::Error> {
         file_path.push_str("/tembo.toml");
 
         let contents = fs::read_to_string(&file_path)?;
-        let config: HashMap<String, InstanceSettings> = toml::from_str(&contents)?;
+        let config: Result<HashMap<String, InstanceSettings>, toml::de::Error> =
+            toml::from_str(&contents);
+
+        match config.clone() {
+            Ok(i) => i,
+            Err(error) => {
+                tui::error(&format!("{}", error));
+                return Ok(());
+            }
+        };
 
         // Validate the config
-        match validate_config(config, verbose) {
+        match validate_config(config.unwrap(), verbose) {
```

Comment:
Is this unwrap definitely safe?

Reply:
Yes I there is a match here for it: https://github.com/tembo-io/tembo/pull/601/files#diff-ee5857f9bcdd882fa5af416c3a7f17c535be574357b8cf55b09945fda09407ddR55